File: modul6/Python/PRAK603-2310817210029-Putra_Whyra_Pratama_Setiawan.py
import numpy as np
a, b = map(int, input().split( ))
baris1 = []
baris2 = []
hasil = []
if(a == b):
    baris1 = np.array(list(map(int, input().split( ))))
    baris2 = np.array(list(map(int, input().split( ))))
    hasil = np.multiply(baris1, baris2)
    print(*hasil)
else:
    print("Jumlah tidak sama")

File: modul6/C/PRAK603-2310817210029-Putra_Whyra_Pratama_Setiawan.c
#include <stdio.h>
int main() {
    int a, b;
    scanf("%d %d", &a, &b);
    int masukkan[2][a], hasil[a];
    if(a == b) {
        for(int i = 0;i < 2;i++) {
            for(int j = 0;j < a;j++) {
                scanf("%d", &masukkan[i][j]);
            }
        }
        for(int i = 0;i < a;i++) {
        hasil[i] = masukkan[0][i] * masukkan[1][i];
        printf("%d ", hasil[i]);
        }
    } else {
        printf("Jumlah tidak sama");
    }
    
}